blocks/form/gdpr.css

```diff
@@ -1,76 +1,34 @@
-.gdpr form input {
-  max-width: 100%;
-  width: 100%;
-}
-
-.gdpr form label {
-  font-size: var(--body-font-size-s);
-}
-
-.gdpr form {
+.gdpr-encrypt form {
   width: 100%;
 }
 
-.gdpr .button-container {
-  display: flex;
-}
-
-.feedback button.button,
-.gdpr form button.button {
-  background-color: var(--c-dark-plum);
-  color: var(--c-light-teal);
-  border-width: thin;
-  border-style: solid;
-  border-color: var(--c-dark-plum);
-  border-image: initial;
-  border-radius: 36px;
-  font-size: 12px;
-  text-align: center;
-  font-weight: 700;
-  position: relative;
-  display: inline-block;
+.gdpr-encrypt form input {
   max-width: 100%;
-  line-height: 36px;
-  text-transform: uppercase;
-  padding: 0 24px;
+  width: 100%;
```

Comment:
also this one I think is redundant

Reply:
correct and removed, but it seems max-width is still needed, kept that one